ldaprc

```diff
@@ -2,4 +2,4 @@ BINDDN  cn=admin,dc=ldap,dc=ldap2pg,dc=docker
 BASE    dc=ldap,dc=ldap2pg,dc=docker
 TLS_REQCERT allow
 NETWORK_TIMEOUT 5
-REFERRALS on
+REFERRALS off
```

Comment:
Do we need this line since the default is off/false?

Reply:
The versionned ldaprc is here just for testing. This is alsow a showcase of ldaprc. Given @datinho added it, let's keep it for a while.